modules/ROOT/pages/installing-kubernetes.adoc

```diff
@@ -0,0 +1,264 @@
+. Deploy the Sync Gateway cluster using the deployment controller file.
++
+[source,console]
+----
+kubectl create -f sgw-deployment.yaml
+----
+If successful, you will see the following.
++
+[source,console]
+----
+deployment.extensions "sync-gateway" created
+----
+. You can check the status of the deployment with the following command until all the pods corresponding to the Sync Gateway are in the "Ready" state and the status is "Running".
++
+[source,console]
+----
+kubectl get pods
+----
++
+If successful, you will see the following.
++
+[source,console]
+----
+NAME                                 READY     STATUS    RESTARTS   AGE
+cb-example-0000                      1/1       Running   0          3d
+cb-example-0001                      1/1       Running   0          3d
+cb-example-0002                      1/1       Running   0          3d
+couchbase-operator-fd8db588b-9fzsw   1/1       Running   1          3d
+sync-gateway-7474f5df4b-c29xw        1/1       Running   2          18m
+sync-gateway-7474f5df4b-p98sq        1/1       Running   0          18m
+----
+
+== Deploying a Load Balancer
+
+In a production deployment, you will likely have one or more Sync Gateway nodes fronted by a xref:load-balancer.adoc[load balancer].
+
+You will deploy the load balancer using the https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/[Kubernetes Load Balancer service].
+The load balancer service provides an externally accessible IP address and routes traffic to the right ports in the cluster.
+
+Follow these steps to deploy a load balancer in front of the Sync Gateway cluster.
+
+. Create a new file called *sgw-load-balancer.yaml* with the following.
++
+[source,yaml]
+----
+kind: Service
+apiVersion: v1
+metadata:
+  name: sgw-load-balancer <1>
+spec:
+  selector:
+    app: sync-gateway <2>
+  ports:
+  - protocol: TCP
+    port: 4984 <3>
+    targetPort: 4984
+  type: LoadBalancer
+----
+<1> `metadata.name`: the name of the load balancer is "sgw-load-balancer".
+<1> `spec.selector.app`: this value corresponds to the pods targeted by the load balancer.
+In this case, it targets any pods with the `app=sync-gateway` label which are the Sync Gateway nodes - this corresponds to what was specified in the deployment yaml file.
+<1> `spec.ports[].targetPort`: the load balancer service targets ports 4984 on the Sync Gateway cluster.
+This is the Sync Gateway port corresponding to the xref:rest-api.adoc[REST API].
+For security purposes, it is recommended that you do not expose the admin port (4985) over the Internet.
+. Deploy the load balancer.
++
+[source,console]
+----
+kubectl create -f sgw-load-balancer.yaml
+----
+If successful, you will see the following.
++
+[source,console]
+----
+service "sgw-load-balancer" created
+----
+. Verify the status of the service creation with the following.
++
+[source,console]
+----
+kubectl get services
+----
+If successful, you will see the following.
```

Comment:
Again, do we want to point out what the user is looking for (i.e. `sgw-load-balancer`)?

Reply:
```suggestion
If successful, you will see a new service corresponding to the load balancer.
In the sample output below, we have the `sgw-load-balancer` service.
```

Reply:
👍